src/hooks/use-update-user.tsx

```diff
@@ -18,6 +18,7 @@ const useUpdateUser = (userId: string) => {
       severity: snackbarVariants.success,
       message: 'editProfilePage.profile.successMessage'
     })
+    window.location.reload()
```

Comment:
use `react-router` tools to reload the page

Reply:
Thank you for advice! Fixed